ui-spacetimechart/src/components/OccupancyBlocLayer.tsx

```diff
@@ -0,0 +1,39 @@
+import { useCallback } from 'react';
+
+import { useDraw } from '../hooks/useCanvas';
+import { type DrawingFunction } from '../lib/types';
+
+export type OccupancyBloc = {
+  timeStart: number;
+  timeEnd: number;
+  spaceStart: number;
+  spaceEnd: number;
+};
+
+export type OccupancyBlocLayerProps = {
+  occupancyBlocs: OccupancyBloc[];
+};
+
+const SIGNALS = ['#CAEDDB', '#FFD4D8', '#FFEABF'];
```

Comment:
could you export well named variables from `/src/lib/consts.ts` ?

Reply:
I'd personally prefer to keep these in this file, since it's really only related to the occupancy layer and not used by anything else. But maybe I'm just not used to the existing practices in osrd-ui.

Reply:
I think that `occupancySemaphore` is far more readable than `#FFD4D8`, from now we have three colors for three signals, in the future this will grow

but if you think it's better that way we can keep it in mind and see later

Reply:
My reaction was about the bit to move this to a separate file only.

I agree that the index is not very descriptive, maybe an enum would be more explicit? Also see related discussion in https://github.com/OpenRailAssociation/osrd-ui/pull/619#discussion_r1791930112.

Reply:
ah yes, np, keeping it in this file seems ok, we should just rename it to be more explicit

Reply:
fix !